boards/v2/qemu/qemu_arc/board.yml

```diff
@@ -0,0 +1,10 @@
+board:
+  name: qemu_arc
+  vendor: Synopsys
```

Comment:
since the board is under `qemu`, having vendor: Synopsys is a bit confusing. What is the policy with qemu boards?

Reply:
Fixed

Reply:
fyi, we had this discussion in the past. The ARC qemu is done by Synopsys, and I think not yet (at least then) upstream. That was the reason for keeping Synopsys as vendor.
But, I'm fine with moving under qemu.